rtrlib/rtr/packets.c

```diff
@@ -369,6 +369,11 @@ static bool rtr_pdu_check_size (const struct pdu_header *pdu) {
     //TODO: Check the error msg 0 termination
 
     break;
+  case SERIAL_QUERY:
+  case RESET_QUERY:
+  case RESERVED:
+  default:
```

Comment:
The code change removes the warning messages but the checks are still missing for those PDU types.

We should either:
- Implement the missing checks for the PDUs
- keep the warnings, so that we are reminded that something is missing and will add them in the future